src/modules/45-dashboards/pages/dashboardView/DashboardView.tsx

```diff
@@ -22,21 +26,19 @@ import {
 } from 'services/custom-dashboards'
 import css from './DashboardView.module.scss'
 
-const DASHBOARDS_ORIGIN = 'https://dashboards.harness.io'
 const DashboardViewPage: React.FC = () => {
   const { getString } = useStrings()
   const { includeBreadcrumbs } = useDashboardsContext()
 
   const { accountId, viewId, folderId } = useParams<AccountPathProps & { viewId: string; folderId: string }>()
   const [embedUrl, setEmbedUrl] = React.useState<string>()
-  const [iframeState] = React.useState(0)
+  const [dashboardFilters, setDashboardFilters] = useQueryParamsState<string | undefined>('filters', undefined)
   const history = useHistory()
-  const query = location.href.split('?')[1]
 
-  const signedQueryUrl: string = useMemo(
-    () => `/embed/dashboards-next/${viewId}?embed_domain=${location.host}&${query}`,
-    [viewId, query]
-  )
+  const signedQueryUrl: string = useMemo(() => {
+    const filters = `&${dashboardFilters}` ?? ''
```

Comment:
I'm pretty sure this will never hit the `?? ''` as the string literal will always have at least the `&`
```suggestion
    const filters = dashboardFilters ? `&${dashboardFilters}` : ''
```

Reply:
🤦  One of those ones where I added in the '&' to fix a bug and was so relieved I didn't replace the nullish coalescing operation